kaal-presentation/build.gradle.kts

```diff
@@ -56,10 +61,12 @@ dependencies {
     implementation(Dependencies.Kotlin.coroutinesCore)
     implementation(Dependencies.Kotlin.coroutinesAndroid)
 
-
     implementation(Dependencies.Android.appCompat)
     api(Dependencies.Android.lifecycleExtension)
     api(Dependencies.Android.viewModelKtx)
+    api(Dependencies.Android.recyclerView)
+    api(Dependencies.Android.viewPager2)
```

Comment:
To reduce transitive dependencies is better to define them only as `compileOnly`.